parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnWriterV1.java

```diff
@@ -57,6 +58,7 @@
   private ValuesWriter dataColumn;
   private int valueCount;
   private int valueCountForNextSizeCheck;
+  private BloomFilterOpts opts;
```

Comment:
Could we rename 'opts' to 'bloomFilterOpts'  to increase readability along the code?